mainwindow.cpp

```diff
@@ -52,9 +121,60 @@ MainWindow::MainWindow(QWidget *parent) : KMainWindow(parent) {
 }
 
 
-void MainWindow::handleTerminalActivity(Terminal *terminal) {
+void MainWindow::createMenuActions()
+{
+    newTabAct = new QAction(tr("&New Tab"), this);
+    newTabAct->setShortcut(QKeySequence(tr("Ctrl+T")));
```

Comment:
shortcuts already set via createShortcuts

this creates a warning saying there is a shortcut ambiguity